ini/ini_map.ts

```diff
@@ -0,0 +1,740 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+/** Options for providing formatting marks. */
+export interface FormattingOptions {
+  /** The character used to assign a value to a key; defaults to '='. */
+  assignment?: string;
+  /** Character(s) used to break lines in the config file; defaults to '\n'. Ignored on parse. */
+  lineBreak?: string;
+  /** Mark to use for setting comments; expects '#', ';', '//', defaults to '#' unless another mark is found. */
+  commentChar?: "#" | ";" | "//";
+  /** Use a plain assignment char or pad with spaces; defaults to false. Ignored on parse. */
+  pretty?: boolean;
+  /** Filter duplicate keys from INI string output; defaults to false to preserve data parity. */
+  deduplicate?: boolean;
+}
+
+/** Options for parsing INI strings. */
+export interface ParseOptions {
+  /** The character used to assign a value to a key; defaults to '='. */
+  assignment?: FormattingOptions["assignment"];
```

Comment:
This option might be unnecessary. @aaronhuggins Have you ever seen ini file using non-`=` assignment mark?

Reply:
I've skimmed a few documents detailing the INI format, and none mention an assignment mark other than `=`. I think it's safe for us to remove this option.